homeassistant/components/aprilaire/strings.json

```diff
@@ -24,6 +24,44 @@
         "name": "Thermostat"
       }
     },
+    "number": {
+      "dehumidification_setpoint": {
+        "name": "Dehumidification setpoint"
+      }
+    },
+    "select": {
+      "air_cleaning_event": {
+        "name": "Air cleaning event",
+        "state": {
+          "off": "[%key:common::state::off%]",
+          "event_clean": "Event clean (3 hour)",
+          "allergies": "Allergies (24 hour)"
+        }
+      },
+      "air_cleaning_mode": {
+        "name": "Air cleaning mode",
+        "state": {
+          "off": "[%key:common::state::off%]",
+          "constant_clean": "Constant clean",
+          "automatic": "Automatic"
+        }
+      },
+      "fresh_air_event": {
+        "name": "Fresh air event",
+        "state": {
+          "off": "[%key:common::state::off%]",
+          "3hour": "3 hour event",
+          "24hour": "24 hour event"
+        }
+      },
+      "fresh_air_mode": {
+        "name": "Fresh air mode",
+        "state": {
+          "off": "[%key:common::state::off%]",
+          "automatic": "Automatic"
+        }
```

Comment:
If we only have on or off, what about a switch?

Reply:
It's not really an on/off. It's off or automatic. Automatic isn't really _on_, it's more of a "let the system decide what to do". I think a switch would be confusing. It would also over complicate the code because it would be one of 4 of the same type of entity to be different. I'd like to keep it as is.

Reply:
I see so this is the ventilation mode (aka fresh air control) - which is not the same as the fan mode exposed in the climate entity.  This is different than a "ventilate now" switch that some hvacs have. While I proposed a switch in my initial comment, I do think a select is clearer.